packages/main/src/Carousel.ts

```diff
@@ -733,7 +714,6 @@ class Carousel extends UI5Element {
 	/**
 	 * The indices of the currently visible items of the component.
 	 * @public
-	 * @readonly
 	 * @since 1.0.0-rc.15
```

Comment:
getters with JSDoc are readonly props so instead of `@returns {Integer[]}` we have to use `@type {Integer[]}` and `@default []`